content/docs/user-guide/experiment-management/cleaning.md

```diff
@@ -0,0 +1,262 @@
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ ├── dac711b [cnn-32]    │ 09:16 AM     │ 0.9152 │ 32               │
+│ ├── 7cd3ae7 [cnn-48]    │ 09:11 AM     │ 0.9131 │ 48               │
+│ ├── ab585b5 [cnn-24]    │ 09:06 AM     │ 0.9135 │ 24               │
+│ ├── 7d51b55 [exp-44136] │ 09:01 AM     │ 0.9151 │ 16               │
+│ └── 7feaa1c [exp-78ede] │ Aug 02, 2021 │ 0.9151 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+Supplying `--all-branches` keeps only the experiments in branch tips. Any
+experiment that's not promoted to a branch is removed this way.
+
+```dvc
+$ dvc exp gc --all-branches
+WARNING: This will remove all experiments except those derived from the workspace and all git branches of the current repo.
+Are you sure you want to proceed? [y/n] y
+Removed 6 experiments. To remove unused cache files use 'dvc gc'.
+```
+
+The resulting `dvc exp show` table is as the following:
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃ Created      ┃    acc ┃ model.conv_units ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ -            │      - │ 64               │
+│ cnn-48                  │ 09:11 AM     │ 0.9131 │ 48               │
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+### Keeping experiments in all tags
+
+- [ ] `--all-tags` example
+
+### Keeping experiments in all commits
+
+- [ ] `--all-commits` example
+
+## Removing Experiments in Remotes
+
+As you push the experiments with `dvc exp push`, remotes may be become cluttered
+with experiment references.
+
+DVC doesn't provide a shortcut for cleaning up the experiments in remotes but
+you can use Git plumbing commands to remove experiment references from remotes.
+
+First get the list of experiments with their hash values.
+
+```dvc
+$ git ls-remote origin 'refs/exps/*'
+```
+
+Then we can use `git push -d` as any other Git reference:
+
+```dvc
+$ git push -d origin refs/exps/path/to/ref
+```
+
+If you want to delete **all** experiments in a remote, you can use a loop:
+
+```dvc
+$ git ls-remote origin 'refs/exps/*' | cut -f 2 | while read exppath ; do
+   git push -d origin "${exppath}"
+done
+```
+
+## Deleting Experiment-Related Objects in DVC Cache
+
+Note that `dvc exp gc` and `dvc exp remove` doesn't delete any objects in the
+DVC <abbr>cache</abbr>. In order to remove the cache objects, e.g. model files,
+intermediate artifacts, etc. related with the experiments, you can use `dvc gc`
+command.
+
+## Deleting All Queued Experiments
+
+When you created experiments to be run in the queue with `--queue` option of
```

Comment:
It should probably be part of the **Removing unnecessary experiments** section (before remote stuff).

Reply:
moved. thank you.